feat(changelog): jobs-fixed-pull-errors-fixed 2025-03-20 (#4668)
scaleway/docs-content: changelog/march2025/2025-03-20-jobs-fixed-pull-errors-fixed.mdx

---
title: Pull errors fixed
status: fixed
date: 2025-03-20
category: serverless
product: jobs
---

> A very low percentage of Job Runs faced pull errors impacting execution. 

Reviewer: ```suggestion
A very low percentage of Job runs faced pull errors impacting execution. 
```

> We identified and fixed the issue on all of our regions.

Reviewer: ```suggestion
We have now identified and fixed the issue across all regions.
```
